src/index.html

```diff
@@ -9,6 +9,10 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
```

Comment:
speed-up must be as modificator but not just additional class
```suggestion
    <div class="stopwatch stopwatch--speed-up">
```